test/parse-dotnet-cli.spec.ts

```diff
@@ -17,39 +17,29 @@ describe('when parsing .NET CLI', () => {
       ).rejects.toThrow('No frameworks were found in project.assets.json');
     });
 
-    it('parse dotnet-cli 2 project and traverse packages', async () => {
-      const projectPath = './test/stubs/dotnet_project';
-      const manifestFile = 'obj/project.assets.json';
-      const expectedTree = JSON.parse(
-        fs.readFileSync('./test/stubs/dotnet_project/expected.json', 'utf-8'),
-      );
-
-      const result = await plugin.inspect(projectPath, manifestFile);
-      expect(result).toEqual(expectedTree);
-    });
-
-    it('parse dotnet-cli 2 with duplicate deps project and traverse packages', async () => {
-      const projectPath = './test/stubs/dotnet_project';
-      const manifestFile = 'obj/project.assets.json';
-      const expectedTree = JSON.parse(
-        fs.readFileSync('./test/stubs/dotnet_project/expected.json', 'utf-8'),
-      );
-      const result = await plugin.inspect(projectPath, manifestFile);
-      expect(result).toEqual(expectedTree);
-    });
-
-    it('parse dotnet-cli project and traverse packages', async () => {
-      const projectPath = './test/stubs/dotnet_p_g';
-      const manifestFile = 'obj/project.assets.json';
-      const expectedTree = JSON.parse(
-        fs.readFileSync('./test/stubs/dotnet_p_g/expected.json', 'utf-8'),
```

Comment:
Deleted this test, as it was pointless. It's supposed to traverse `_packages` but the folder doesn't exist. Another test is already doing it.